Sources/Plasma/PubUtilLib/plInputCore/plInputInterfaceMgr.h

```diff
@@ -153,18 +154,17 @@ class plInputInterfaceMgr : public plSingleModifier
 class plCtrlCmd
 {
     private:
-        char*               fCmd;
+        ST::string          fCmd;
         plInputInterface    *fSource;
 
     public:
         plCtrlCmd(plInputInterface* source)
             : fCmd(), fPct(1.f), fSource(source),
               fControlCode(), fControlActivated(), fNetPropagateToPlayers()
         { }
-        ~plCtrlCmd() { delete [] fCmd; }
 
-        const char* GetCmdString()          { return fCmd; }
-        void SetCmdString(const char* cs)   { delete [] fCmd; fCmd=hsStrcpy(cs); }
+        ST::string GetCmdString() { return fCmd; }
```

Comment:
While we're here....

```suggestion
        ST::string GetCmdString() const { return fCmd; }
```